main.go

```diff
@@ -62,7 +62,7 @@ func main() {
 	klog.InitFlags(flag.CommandLine)
 	flag.Parse()
 
-	logger := zap.New(zap.UseFlagOptions(&opts))
+	logger := zap.New(zap.UseFlagOptions(&opts), zap.JSONEncoder())
```

Comment:
Your change does the work and it's possibly the easiest one. Digging into the documentation I found:
```
[ pipeline-service-exporter] -> go doc sigs.k8s.io/controller-runtime/pkg/log/zap.Options.Encoder
package zap // import "sigs.k8s.io/controller-runtime/pkg/log/zap"

type Options struct {
    // Encoder configures how Zap will encode the output. Defaults to console when
    // Development is true and JSON otherwise
    Encoder zapcore.Encoder
```
As I understand it, if we set the development option to `false` here https://github.com/openshift-pipelines/pipeline-service-exporter/blob/main/main.go#L59 we could achieve the same effect. I haven't tested it though. So the question is if there is specific reason to have that development option enabled? If not disabling it and getting the same result will be the better approach IMO.

Reply:
Bottom line is, if we need that change sooner, I'll lgtm and we can merge it. If it can wait for till the next week, lets wait and see what @gabemontero has to say. In the meantime you can try if my understanding from the documentation is right.

Reply:
Hi @enarha, yes the alternative is to put development to `true` and that by default uses `json` encoder. I think the only difference we'll have by development `true` is we'll have stacktraces on errors rather than on warning. 
Ans sure, we can wait for @gabemontero to chime in.

Reply:
Yes, I can confirm it does have json encoder with `Development: false`

Reply:
I'm fine with stack traces on errors and not on warnings

Let's go with @enarha 's suggestion of no longer setting Development to true @ramessesii2

Reply:
Sure, I'm fine with it.